deploy/chaos-exporter.yaml

```diff
@@ -1,47 +1,31 @@
 ---
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: v1
+kind: Pod
 metadata:
-  name: chaos-exporter
-  namespace: default 
-  labels:
-    app: chaos-exporter
+  name: chaos-monitor
+  namespace: litmus
 spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: chaos-exporter
-  template:
-    metadata:
-      labels:
-        app: chaos-exporter 
-    spec:
-      serviceAccountName: chaos-operator
-      containers:
-      - name: chaos-exporter
-        #image: ksatchit/sample-chaos-exporter:ci 
-        image: litmuschaos/chaos-exporter:ci 
-        imagePullPolicy: Always
-        env:
-          - name: CHAOSENGINE
-            value: engine-nginx
-
-          - name: APP_UUID
-            value: "3f2092f8-6400-11e9-905f-42010a800131" 
- 
-        ports:
-        - containerPort: 8080
+  containers:
+  - image: litmuschaos/chaos-exporter:ci
+    imagePullPolicy: Always
+    name: chaos-monitor
+    ports:
+    - containerPort: 8080
+      name: metrics
+      protocol: TCP
+  serviceAccount: litmus
+  serviceAccountName: litmus
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: chaos-exporter
-  namespace: default
+  name: chaos-monitor
```

Comment:
If the chaos-monitor is pod then how this service is connected with that pod?

Reply:
Yes, have made the changes in the service yaml also.

Reply:
we will make it deployment

Reply:
I think here we can keep the exporter as deployment and in the pages repo we will add the exporter as a sidecar with operator